server/src/api/invitations/invitations.controller.ts

```diff
@@ -8,6 +9,14 @@ import { daysAgo } from '../../utils/date';
 // should FRONTEND_BASE_URL and CORS_ORIGIN be same variable?
 const BASE_URL = process.env.FRONTEND_BASE_URL!;
 
+export const deleteOldInvitations = async (
+  _: Context,
+  next: () => Promise<any>,
+) => {
+  await Invitation.query().delete().where('updatedAt', '<', daysAgo(30));
+  await next();
+};
```

Comment:
```suggestion
const deleteOldInvitations = () =>
  Invitation.query().delete().where('updatedAt', '<', daysAgo(30));
```

I would rather keep this inside the controller, and just add `await deleteOldInvitations();` at the top of `postInvitation()`

Reply:
I contemplated whether to use a middleware or not - changed it to the suggested way